sql/insert.go

```diff
@@ -92,25 +111,27 @@ func (p *planner) Insert(n *parser.Insert) (planNode, error) {
 	if err := rows.Err(); err != nil {
 		return nil, err
 	}
-	if err := p.db.Run(&b); err != nil {
+	if err := p.db.Txn(func(txn *client.Txn) error {
```

Comment:
Why did you add a transaction here? We want to implement transactions at a higher layer to include many inserts, etc